dev-python/setuptools_rust/setuptools_rust-0.12.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+MY_PN="${PN/_/-}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="a plugin for setuptools to build Rust Python extensions"
+HOMEPAGE="https://github.com/PyO3/setuptools-rust"
+SRC_URI="mirror://pypi/${PN::1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~riscv ~x86"
```

Comment:
ditto with keywords here. only add `~amd64`, the bot check will become green.

Reply:
Right, of course. Done.